src/ListTabulator/index.ts

```diff
@@ -810,18 +1037,18 @@ export default class ListTabulator {
       return;
     }
 
-    Caret.focus(itemContent, atStart);
+    focus(itemContent, atStart);
   }
 
   /**
    * Get out from List Tool by Enter on the empty last item
    *
    * @returns {void}
    */
-    getOutOfList(): void {
-      this.currentItem?.remove();
+  getOutOfList(): void {
+    this.currentItem?.remove();
 
-      this.api.blocks.insert();
-      this.api.caret.setToBlock(this.api.blocks.getCurrentBlockIndex());
-    }
+    this.api.blocks.insert();
+    this.api.caret.setToBlock(this.api.blocks.getCurrentBlockIndex());
```

Comment:
`insert` return as Block that you can pass to the `carent.setToBlock()`

Reply:
as i can see, `insert` returns block api which has only string `id` (can't see the index)
![image](https://github.com/user-attachments/assets/9c673da3-c1e0-4247-b37a-5eda749b086c)

Reply:
you don't need index, `caret.setToBlock()` can accept a BlockApi as well